### ISSUE TITLE
Update YCBA logo in IIIF viewer

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#356]
It was suggested at the 3/18 meeting that we update the YCBA logo in the viewer. 
Julie Fry, YCBA Head of Design, suggested white logo with transparent background (which would work well with accessibility requirements as long as the viewer's default setting is on the black/dark gray background).
@yulgit1 Are there any concerns about pushing it to prod?

Current/old logo:
![YCBA Universal Viewer with old YCBA logo](https://user-images.githubusercontent.com/4093644/159347739-e7cddb15-af20-44d1-a8d3-cd2a53877a9b.jpg)

Proposed/new logo:
![YCBA Universal Viewer with white png logo](https://user-images.githubusercontent.com/4093644/159348213-867873de-83d1-4649-a94c-62e3675570b9.png)

**Comment by yulgit1** [comment]
@edgartdata  95% of the time people will be viewing from our collection page, but there is no guarantee the user of our manifests will be using UV or mirador3 in dark mode.

**Comment by edgartdata** [comment]
Will use a non-transparent white logo on dark gray background.

**Comment by edgartdata** [comment]
@edgartdata to check next week

**Comment by yulgit1** [comment]
@edgartdata Please see in collections-test where UV should be up now.
